addons/hr_recruitment/models/ir_ui_menu.py

```diff
@@ -9,7 +9,9 @@ class IrUiMenu(models.Model):
 
     def _load_menus_blacklist(self):
         res = super()._load_menus_blacklist()
-        if self.env.user.has_group('hr_recruitment.group_hr_recruitment_interviewer') and not self.env.user.has_group('hr_recruitment.group_hr_recruitment_user'):
+        if not self.env.user.has_group('hr_recruitment.group_hr_recruitment_interviewer'):
+            res.append(self.env.ref('hr.menu_view_hr_job').id)
+        elif self.env.user.has_group('hr_recruitment.group_hr_recruitment_interviewer') and not self.env.user.has_group('hr_recruitment.group_hr_recruitment_user'):
             res.append(self.env.ref('hr_recruitment.menu_hr_job_position').id)
         elif self.env.user.has_group('hr_recruitment.group_hr_recruitment_user'):
             res.append(self.env.ref('hr_recruitment.menu_hr_job_position_interviewer').id)
```

Comment:
It could be an opportunity to make this a bit more readable since , what do you think?
```suggestion
        is_interviewer = self.env.user.has_group('hr_recruitment.group_hr_recruitment_interviewer')
        is_user = self.env.user.has_group('hr_recruitment.group_hr_recruitment_user')
        if not is_interviewer:
            res.append(self.env.ref('hr.menu_view_hr_job').id)
        elif is_interviewer and not is_user:
            res.append(self.env.ref('hr_recruitment.menu_hr_job_position').id)
        else:
            res.append(self.env.ref('hr_recruitment.menu_hr_job_position_interviewer').id)
```
could even be:
```py
        is_interviewer = self.env.user.has_group('hr_recruitment.group_hr_recruitment_interviewer')
        is_user = self.env.user.has_group('hr_recruitment.group_hr_recruitment_user')
        if not is_interviewer:
            return res + [self.env.ref('hr.menu_view_hr_job').id]
        if is_interviewer and not is_user:
            return res + [self.env.ref('hr_recruitment.menu_hr_job_position').id]
        return res + [self.env.ref('hr_recruitment.menu_hr_job_position_interviewer').id]
```